controllers/install/lookout_controller.go

```diff
@@ -173,9 +186,15 @@ func generateLookoutInstallComponents(lookout *installv1alpha1.Lookout, scheme *
 	// if err := controllerutil.SetOwnerReference(lookout, serviceAccount, scheme); err != nil {
 	// 	return nil, err
 	// }
-	job := createLookoutMigrationJob(lookout)
-	if err := controllerutil.SetOwnerReference(lookout, job, scheme); err != nil {
-		return nil, err
+	var job *batchv1.Job
```

Comment:
Test coverage dropped by 3%.  Could you maybe add a test case where you enable lookout migrations and one where you don't?

Reply:
sure